README.md

```diff
@@ -1,85 +1,133 @@
-For more details, please see our [Code of Conduct](https://github.com/djangoindia/djangoindia.org/blob/main/CODE_OF_CONDUCT.md).
-
-## Contributing Guidelines
-
-We welcome contributions from everyone, whether you're fixing bugs, adding new features, or improving documentation. To contribute, please fork the repository, create a new branch for your changes, and submit a pull request. Make sure to follow our coding standards and include tests where applicable. We also encourage you to discuss significant changes by opening an issue before starting development. Your contributions help us build a stronger community—thank you for being a part of Django India!
-For more details, please see our [Contributing Guidelines](https://github.com/djangoindia/djangoindia.org/blob/main/contributing.md)
-
-## License
-
-This project is licensed under the MIT License, one of the most permissive and widely used open-source licenses.This means that you can integrate this project into your applications, modify it to fit your needs, and even distribute it as part of a commercial product without any concern for legal restrictions.
-For more details, please see our [License](https://github.com/djangoindia/djangoindia.org/blob/main/LICENSE)
-
-
-## Getting Started
-1. **Fork the repository:**
-
-    - Read the [documentation](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/fork-a-repo) to learn about forking a repository.
-
-2. **Clone the repository:**
-
-    ```
-    git clone https://github.com/<your_username>/djangoindia.org
-    ```
-
-- [Docker](https://github.com/djangoindia/djangoindia.org/blob/main/contributing.md#without-docker)
-- [Manual](https://github.com/djangoindia/djangoindia.org/blob/main/contributing.md#with-docker)
-
-
+For more details, please see our [Code of Conduct.](https://github.com/djangoindia/djangoindia.org/blob/main/CODE_OF_CONDUCT.md)
 
 ##  Get Involved and Make a Difference
 
 
-Join our community and help build **Zango**. Here's how you can get involved:
+Join our community. Here's how you can get involved:
 
 - **Star the Repo:** Show your support by giving us a star! ⭐️
 - **Spread the Word:** Share Django India with your colleagues and friends. 📣
 - **Report Issues:** Notice something not quite right? Let us know by creating an issue. Your feedback is invaluable! 🐛
 - **Contribute Code:** Dive into open issues and send pull requests to help us squash bugs and implement exciting enhancements. 🛠️
+- **Review Pull requests:** By actively reviewing pull requests, you can help maintain the quality and integrity of the project, while also supporting other contributors and fostering a collaborative environment within the Django India community.🕵️
   
 Together, let's build something incredible! ✨🚀
 
-#### Staying ahead
 
-Star Django India on GitHub and be instantly notified of new releases!
 
-## Website Lookup
+## License
```

Comment:
GitHub and LinkedIn have already been added as socials, which is enough I guess.
A few changes which I would like to suggest such as 
- moving the Licence section to the end of the readme.
- as the `CODE_OF_CONDUCT.md` file is already added, then we do not need to include it in the readme file.
- as it is the official website repository of Django India, we may see frequent contributions in this repository so we can add a stats section that will summarise the recent activity in the repository (as given in image below)
![image](https://github.com/user-attachments/assets/3ca5af29-fd23-48e1-bd7c-2fedb8ce7700)

Reply:
> as the CODE_OF_CONDUCT.md file is already added, then we do not need to include it in the readme file.

Its just a short summary which gives a rough rules ideas for our community. looks fine to me.

> as it is the official website repository of Django India, we may see frequent contributions in this repository so we can add a stats section that will summarise the recent activity in the repository (as given in image below)

looks good , where can i find it?

Reply:
You can find it at the bottom of this page - https://github.com/alexkates/hashnode-next

Reply:
asking for login with github, not sure if i want to do that. so skipping for now.